doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -159,10 +159,14 @@ def default_imagestream_base_name(version: str, runtime: Runtime) -> str:
 
 def assembly_imagestream_base_name(runtime: Runtime) -> str:
     version = runtime.get_minor_version()
-    if runtime.assembly == 'stream' and runtime.assembly_type is AssemblyTypes.STREAM:
-        return default_imagestream_base_name(version, runtime)
+    return assembly_imagestream_base_name_generic(version, runtime.assembly, runtime.assembly_type)
+
+
+def assembly_imagestream_base_name_generic(minor_version, assembly_name, assembly_type):
```

Comment:
```suggestion
def assembly_imagestream_base_name_generic(version, assembly_name, assembly_type):
```
let's change this var name to "version" to be clear